client/src/pages/layout/Navbar.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Navbar as NavbarB, NavDropdown, Nav } from "react-bootstrap";
+import "./Navbar.css";
+const Navbar = () => {
+  return (
+    <NavbarB className="navbar-custom" bg="transparent" expand="lg">
+      <NavbarB.Brand className="navbar-brand-text" as={Link} to="/">
+        idiomia
+      </NavbarB.Brand>
+      <NavbarB.Toggle aria-controls="basic-navbarB-nav" />
+      <NavbarB.Collapse id="basic-navbarB-nav">
+        <Nav className="ml-auto">
+          <NavDropdown
+            className="navbar-dropdown"
+            title="HOME"
+            id="navbar-dropdown"
+          >
+            <NavDropdown.Item as={Link} to="/">
+              About
```

Comment:
Have you defined pages for About/Faqs/Contact? If you want these pages - lets make tickets for them in the project board.  Maybe it is better to remove these for now and then add them in when we have the pages implemented

Reply:
Actually we have discussed About page with Ammar. Yes I can remove them for now.